src/lib/holocene/code-block.svelte

```diff
@@ -56,6 +57,7 @@
   export { className as class };
   export let editable = false;
   export let inline = false;
+  export let grow = true;
```

Comment:
I don't love adding a prop for a single css class. Why don't we want the code block to grow in some instances? We are already exposing a `class` prop, can't we just do `class="grow-0"` where we don't want this to grow?

Reply:
If it always grows, then the input looks like this:
<img width="1728" alt="Screenshot 2024-12-04 at 10 38 34 AM" src="https://github.com/user-attachments/assets/c552b027-297d-4c69-a19b-ef66aa63c9d4">

The class prop is on the codeblock div itself, not on the container that needs to grow.

Reply:
Ohh gotcha. I think we can probably just add `grow-0` to the top level div of the code block component. I don't see a reason why we'd ever want the code block wrapper to be bigger than the codeblock itself, and this probably only occurs if the codeblock is in a flex element 🤷

Reply:
👍 good call